docs/pages/docker.md

```diff
@@ -1,3 +1,45 @@
 # Run with Docker
 
-TODO: Add instructions on how to run Aana Apps with Docker.
\ No newline at end of file
+We provide a docker-compose configuration to run the application in a Docker container in [Aana App Template](https://github.com/mobiusml/aana_app_template/?tab=readme-ov-file#running-with-docker).
+
+Requirements:
+
+- Docker Engine >= 26.1.0
+- Docker Compose >= 1.29.2
+- NVIDIA Driver >= 525.60.13
+
+You can edit the [Dockerfile](https://github.com/mobiusml/aana_app_template/Dockerfile.yaml) to assemble the image as you desire and
```

Comment:
I don't think that link will work. It should have `blob/main` like https://github.com/mobiusml/aana_app_template/blob/main/pyproject.toml

Reply:
fixed all of them